src/nestedArray/types.ts

```diff
@@ -53,13 +62,21 @@ const DTYPE_TYPEDARRAY_MAPPING: { [A in DtypeString]: TypedArrayConstructor<Type
   '>u4': Uint32Array,
   '>i4': Int32Array,
   '>f4': Float32Array,
+  '>f2': globalThis.Float16Array,
   '>f8': Float64Array
-};
+});
 
 
 export function getTypedArrayCtr(dtype: DtypeString) {
-  const ctr = DTYPE_TYPEDARRAY_MAPPING[dtype];
+  const ctr = DTYPE_TYPEDARRAY_MAPPING()[dtype];
```

Comment:
I personally would've just kept it simple and require Float16Array to be defined prior to importing Zarr. Of course the performance differences are negligible, but it's the kind of thing where I tend to be strict rather than lenient

Reply:
I tried this, but I think due to how jest bundles/transpiles modules I am unable to set `Float16Array` globally in our test suite prior to importing zarr.js.

Reply:
Ok... actually I can do a dynamic import in the tests.